libs/ui-watt/src/lib/foundations/typography/_typography.import.scss

```diff
@@ -14,97 +14,155 @@
+    'line-height': 42px,
+  ),
+  'xl': (
+    'font-size': 26px,
+    'line-height': 36px,
+  ),
+  'l': (
+    'font-size': 20px,
+    'line-height': 28px,
+  ),
+  'm': (
+    'font-size': 16px,
+    'line-height': 24px,
+  ),
+  's': (
+    'font-size': 14px,
+    'line-height': 20px,
+  ),
+);
+
+@mixin font-size($size) {
+  $sizes: map.get($font-sizes, $size);
+  font-size: utils.rem(map.get($sizes, 'font-size'));
+  line-height: utils.rem(map.get($sizes, 'line-height'));
+}
+
+@mixin font-weight($weight) {
+  font-weight: map.get($font-weights, $weight);
+}
+
 @mixin watt-headline-1 {
   color: var(--watt-color-primary-dark);
-  font-size: 2.25rem;
-  font-weight: 600;
+  @include font-size('xl');
+  @include font-weight('bold');
   letter-spacing: 0;
-  line-height: 1.5;
-  text-transform: uppercase;
+
+  @include breakpoints.media('>=Large') {
+    @include font-size('xxl');
+  }
 }
 
 @mixin watt-headline-2 {
   color: var(--watt-color-primary-dark);
-  font-size: 2rem;
-  font-weight: 600;
+  @include font-size('l');
+  @include font-weight('bold');
   letter-spacing: 0;
-  line-height: 1.5;
-  text-transform: uppercase;
+
+  @include breakpoints.media('>=Large') {
+    @include font-size('xl');
+  }
 }
 
 @mixin watt-headline-3 {
   color: var(--watt-color-primary-dark);
-  font-size: 1.75rem;
-  font-weight: 600;
+  @include font-size('m');
+  @include font-weight('bold');
   letter-spacing: 0;
-  line-height: 1.5;
-  text-transform: uppercase;
 }
 
 @mixin watt-headline-4 {
   color: var(--watt-color-primary-dark);
-  font-size: 1.5rem;
-  font-weight: 600;
+  @include font-size('m');
+  @include font-weight('bold');
   letter-spacing: 0;
-  line-height: 1.5;
-  text-transform: uppercase;
 }
 
 @mixin watt-headline-5 {
   color: var(--watt-color-primary-dark);
-  font-size: 1.25rem;
-  font-weight: 600;
+  @include font-size('m');
+  @include font-weight('bold');
   letter-spacing: 0;
-  line-height: 1.5;
-  text-transform: uppercase;
 }
 
 @mixin watt-text-l {
-  color: var(--watt-color-primary-dark);
-  font-size: 1.125rem;
-  font-weight: 600;
+  @include font-size('l');
+  @include font-weight('regular');
   letter-spacing: 0;
-  line-height: 1.5;
 }
 
 @mixin watt-text-m {
-  color: var(--watt-color-primary-dark);
-  font-size: 1rem;
-  font-weight: 400;
+  @include font-size('m');
+  @include font-weight('regular');
   letter-spacing: 0;
-  line-height: 1.5;
 }
 
 @mixin watt-text-s {
-  color: var(--watt-color-primary-dark);
-  font-size: 0.875rem;
-  font-weight: 400;
+  @include font-size('s');
+  @include font-weight('regular');
   letter-spacing: 0;
-  line-height: 1.5;
 }
 
-@mixin watt-text-xs {
-  color: var(--watt-color-primary-dark);
-  font-size: 0.75rem;
-  font-weight: 400;
-  letter-spacing: 0;
-  line-height: 1.5;
+@mixin watt-text-l-highlighted {
+  @include font-size('l');
+  @include font-weight('bold');
+}
+
+@mixin watt-text-m-highlighted {
+  @include watt-text-m;
+  @include font-weight('bold');
```

Comment:
suggestion: Add `letter-spacing: 0;` similar to the other mixins

Reply:
The highlighted mixins are using their not highlighted mixin which includes the letter-spacing